code/__DEFINES/melee.dm

```diff
@@ -11,4 +11,4 @@
 #define MARTIALART_WRESTLING "wrestling"
 
 /// The number of hits required to crit a target
-#define HITS_TO_CRIT(damage) round(100 / damage, 0.1)
+#define HITS_TO_CRIT(damage) round(MAX_LIVING_HEALTH / damage, 0.1)
```

Comment:
r u sure this works it seems to be above where it's actually defined

Reply:
it compiled and i'm almost afraid to ask how

Reply:
> r u sure this works it seems to be above where it's actually defined

tested it. somehow it works.